src/javascript/javascript-file.ts

```diff
@@ -0,0 +1,122 @@
+import { js } from 'js-beautify';
+import { TextFileOptions, TextFile, Project } from '..';
```

Comment:
I think this should fix the build errors:
```suggestion
import { Project } from '../project';
import { TextFile, TextFileOptions } from '../textfile';
```